lib/internal/fs/promises.js

```diff
@@ -266,9 +267,8 @@ async function handleFdClose(fileOpPromise, closeFunc) {
 }
 
 async function fsCall(fn, handle, ...args) {
-  if (handle[kRefs] === undefined) {
-    throw new ERR_INVALID_ARG_TYPE('filehandle', 'FileHandle', handle);
-  }
+  assert(handle instanceof FileHandle,
```

Comment:
Let's avoid `instanceof`, a property lookup is more efficient I believe 
```suggestion
  assert(handle[kRefs] !== undefined,
```